web/src/app/services/HeartbeatMonitorForm.tsx

```diff
@@ -12,19 +13,19 @@ function clampTimeout(val: string): number | string {
   // need to have the min be 1 here so you can type `10`
   return Math.min(Math.max(1, num), 9000)
 }
-interface Value {
+export interface Value {
   name: string
-  timeoutMinutes: [number, string]
+  timeoutMinutes: [number, string] | number
```

Comment:
I think this should always be `number`?

I wasn't sure on `[number, string]` being needed (looks like it came from #742)

It's used as such:

![Screen Shot 2022-07-19 at 12 34 50 PM](https://user-images.githubusercontent.com/11381794/179833915-34754c73-9b5d-4849-968a-bc727498d952.png)

though our schema has it assigned as only an int:

![Screen Shot 2022-07-19 at 12 34 58 PM](https://user-images.githubusercontent.com/11381794/179833967-299dc27a-0217-46c6-9b0c-6df124cfb55f.png)